src/api/digitalSpecimen/GetDigitalSpecimenAnnotations.ts

```diff
@@ -0,0 +1,43 @@
+/* Import Dependencies */
+import axios from 'axios';
+
+/* Import Types */
+import { Annotation } from 'app/types/Annotation';
+import { JSONResultArray } from 'app/Types';
+
+/* Import Exceptions */
+import { NotFoundException } from 'app/Exceptions';
+
+
+/**
+ * Function for fetching a digital specimen's annotations
+ * @param handle The identifier of the digital specimen
+ * @returns Array of Annotations
+ */
+const GetDigitalSpecimenAnnotations = async({ handle } : { handle: string }) => {
+    let digitalSpecimenAnnotations: Annotation[] = [];
+
+    if (handle) {
+        try {
+            const result = await axios({
```

Comment:
Is does not really matter, both would have the same effect.
I liked doing this as all the other stuff that defines the request is also in there.
But if you have a deep craving for it, it can be changed of course

Reply:
do what makes your heart sing, tom